massa-node/src/main.rs

```diff
@@ -473,6 +477,26 @@ async fn launch(
         }
     }
 
+    // Event cache thread
+    let event_cache_config = EventCacheConfig {
+        event_cache_path: SETTINGS.execution.event_cache_path.clone(),
+        max_event_cache_length: SETTINGS.execution.event_cache_size,
+        snip_amount: SETTINGS.execution.event_snip_amount,
+        max_event_data_length: MAX_EVENT_DATA_SIZE as u64,
+        thread_count: THREAD_COUNT,
+        // Note: SCOutputEvent call stack comes from the execution module, and we assume
+        //       this should return a limited call stack length
+        //       The value remains for future use & limitations
+        max_call_stack_length: u16::MAX,
+
+        max_events_per_operation: MAX_EVENT_PER_OPERATION as u64,
+        max_operations_per_block: MAX_OPERATIONS_PER_BLOCK as u64,
+        max_events_per_query: MAX_EVENTS_PER_QUERY,
+        tick_delay: Duration::from_millis(EVENT_CACHE_TICK_DELAY),
```

Comment:
the tick delay stuff should be dropped and just use a wait() on the condition variable

Reply:
Fixed